dev-cpp/mvfst/mvfst-2024.11.04.00-r1.ebuild

```diff
@@ -40,8 +40,8 @@ DEPEND="
 
 src_configure() {
 	local mycmakeargs=(
-		-DCMAKE_INSTALL_DIR="$(get_libdir)/cmake/${PN}"
-		-DLIB_INSTALL_DIR="$(get_libdir)"
+		-DCMAKE_INSTALL_MODULE_DIR="$(get_libdir)/cmake/${PN}"
```

Comment:
I think these are the usual ones that GNUInstallDirs and cmake.eclass handle, so you may not need to set them at all, FWIW. (Or maybe you do because libdir and not share?)

Reply:
I'll test whether replacing them in all Facebook packages builds fine, and send in a PR removing them if so. I inherited this from the previous versions of the package and didn't pay it much attention, tbh.